src/include/daos/object.h

```diff
@@ -103,12 +103,13 @@ enum {
 	 *
 	 * NB: it should be smaller than OC_BACK_COMPAT (50)
 	 */
-	DAOS_OC_OIT_V1	= 45,
+	DAOS_OC_OIT_RF0	= 45,
+	DAOS_OC_OIT_RF1	= 46,
+	DAOS_OC_OIT_RF2	= 47,
+	DAOS_OC_OIT_RF3	= 48,
+	DAOS_OC_OIT_RF4	= 49,
```

Comment:
Why need to define so many OIT class? does it waste the reserved object class?

Reply:
for cont_rf we only with 5 - RF0 - RF4, so using 5 OC (adding 4) looks fine?

Reply:
It's fine for me, just not sure whether we will use some of them or not, such as DAOS_OC_OIT_RF4.